### ISSUE TITLE
Collaboration Cycle - Create a health metric dashboard

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43022]
## Problem Statement 
Governance team would like to build a health metric dashboard to track VFS engagement, use and outcomes of the Collaboration Cycle. This would allow us to track trends, recognize patterns and make observations to better understand how the product is used by VFS teams.

How might we create a dashboard that provides helpful, ideally automated metrics on the health of the Collaboration Cycle?

**Proposed metrics for measuring Collaboration Cycle health:** https://vfs.atlassian.net/wiki/spaces/PPST/pages/2237661394/Collaboration+Cycle+-+Create+a+health+metric+dashboard

**Metric for this initiative:** Median length of time it takes a VFS team to move through Collaboration Cycle (kickoff to post-staging)

**9/9/2022 update:** This initiative has been put on hold and moved to the Governance team backlog. Due to resources constraints, in addition to challenges measuring the metric above without manual intervention, the Governance team will be pivoting to initiative #45156. The data set for Collaboration Cycle Staging Review issue tickets is already defined and ready for card building. 

## Hypothesis or Bet
If we build a dashboard that shows metrics on the health of the Collaboration Cycle, then we'll have the information necessary to recognize where improvements need to be made to the Collaboration Cycle process.

## We will know we're done when... ("Definition of Done")
*What requirements does this project need to meet for you to finish this initiative?*
- [ ] Critical Collaboration Cycle metrics are identified with monitoring and exposed through dashboard
- [ ] We have identified and created a data set for obtaining metrics
- [ ] We will have at least 1 automated metric on the health of the Collaboration Cycle
- [ ] We have identified opportunities for automating valuable metrics
- [ ] 

## Known Blockers/Dependencies 
*List any blockers or dependencies for this work to be completed*
Subject matter expertise in Domo and/or DataDog
Training from Analytics and Insights Team

## Projected Launch Date
Determine high level timeline

### Guidance (delete before posting)

_This checklist is intended to be used to help answer, "is my Platform initiative ready for launch?". All of the items in this checklist should be completed, with artifacts linked---or have a brief explanation of why they've been skipped---before launching a given Platform initiative. All links or explanations can be provided in **Required Artifacts** sections. The items that can be skipped are marked as such._

_Keep in mind the distinction between **Product** and **Initiative** --- each Product needs specific supporting documentation, but Initiatives to improve existing Products should reuse existing documentation for that Product.  [VSP Product Terminology](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/product-management/product-terminology.md) for details._

### ... tested?

- [ ] Usability test (_TODO: link_) has been performed, to validate that new changes enable users to do what was intended and that these changes don't worsen quality elsewhere. If usability test isn't relevant for this change, document the reason for skipping it.
    - [ ] ... and issues discovered in usability testing have been addressed.
    * _Note on skipping: metrics that show the impact of before/after can be a substitute for usability testing._
- [ ] End-to-end [manual QA](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/README.md) or [UAT](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/planning/what-is-uat.md) is complete, to validate there are no high-severity issues before launching
- [ ] _(if applicable)_ New functionality has thorough, automated tests running in CI/CD

### ... documented?

- [ ] New documentation is written pursuant to our [documentation style guide](https://vfs.atlassian.net/wiki/spaces/AP/pages/622264362/Style+guide)
- [ ] Product is included in the [List of VSP Products](https://docs.google.com/spreadsheets/d/1Fn2lD419WE3sTZJtN2Ensrjqaz0jH3WvLaBtn812Wjo/edit#gid=0)
    * _List the existing product that this initiative fits within, or add a new product to this list._
- [ ] Internal-facing: there's a [Product Outline](https://vfs.atlassian.net/wiki/spaces/PMCP/pages/1924628490/Product+Outline+Template)
- [ ] External-facing: a [User Guide on Platform Website](https://vfs.atlassian.net/wiki/spaces/AP/pages/1477017691/Platform+website+guidelines) exists for this product/feature tool
- [ ] _(if applicable)_ Post to [#vsp-service-design](https://dsva.slack.com/channels/vsp-service-design) for external communication about this change (e.g. customer-facing meetings)

### ... measurable

- [ ] _(if applicable)_ This change has clearly-defined success metrics, with instrumentation of those analytics where possible, or a reason documented for skipping it.
    * For help, see: [Analytics team](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guide.1586823275.html)
- [ ] This change has an accompanying [VSP Initiative Release Plan](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose).

### When you're ready to launch...
- [ ] Conduct a [go/no-go] (https://vfs.atlassian.net/wiki/spaces/AP/pages/1670938648/Platform+Crew+Office+Hours#Go%2FNo-Go) when you're almost ready to launch. 

### Documentation
* **`PRODUCT_NAME`**: _directory name used for your product documentation_
* **Product Outline**: _link to Product Outline_
* **User Guide**: _link to User Guide_

### Testing

* **Usability test**: _link to GitHub issue, or provide reason for skipping_
* **Manual QA**: _link to GitHub issue or documented results_
* **Automated tests**: _link to tests, or "N/A"_

### Measurement

* **Success metrics**: _link to where success metrics are measured, or where they're defined (Product Outline is OK), or provide reason for skipping_
* **Release plan**: _link to Release Plan ticket_

## TODOs
- [ ] Convert this issue to an epic
- [ ] Add your team's label to this epic
- [ ] Add this initiative to the [Digital Experience Product Board](https://github.com/orgs/department-of-veterans-affairs/projects/306?card_filter_query=label%3Acrew-platform) using [these instructions](https://vfs.atlassian.net/wiki/spaces/AP/pages/2120974387/Platform+Initiative+Guidelines+for+Digital+Experience+Product+Board)

**Comment by shiragoodman** [comment]
Potential metrics to track:

- Median (not average) lifecycle from kickoff to staging
- ideally automated (possibly next iteration)
- MVP for this initiative

Could we get (# of open collab cycle request, completed cc requests) in bigQuery on a monthly basis?
Chelen: suggest going over the metrics that are measurable and able to be automated. Build tooling around new Collab Cycle REquest, not older process.

**Comment by shiragoodman** [comment]
@raywangoctova I have added an update to the Problem Statement section above. I have also moved the Domo training ticket to new initiative #45156, and closed our ticket for building the new data set. Please let me know what other changes need to be made (if any) before we move this to the Future Initiatives pipeline.

**Comment by annekerr49** [comment]
@raywangoctova - Ray could we get some specifics about this activity?

**Comment by shiragoodman** [comment]
Hi Anne @annekerr49 - this was a Governance team initiative for Q3 of 2022, however we moved it to our "Future Initiatives" pipeline due to challenges in bringing in the data set to Domo, and also due to resource limitations with the Analytics & Insights team, a team we were dependent on for Domo support. Instead we pivoted to [this initiative](https://github.com/department-of-veterans-affairs/va.gov-team/issues/45156) and re-launched the [Collab Cycle Dashboard. ](https://va-gov.domo.com/page/1340527018?userId=1646566899) We have a [phase 2 initiative](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47911) planned for later this quarter that will expand the current dashboard.
